sdk/tables/data-tables/src/TablesSharedKeyCredentialPolicy.ts

```diff
@@ -60,70 +60,66 @@ export class TablesSharedKeyCredentialPolicy extends BaseRequestPolicy {
    * @param {WebResource} request
    * @returns {WebResource}
    */
-  protected signRequest(request: WebResourceLike): WebResource {
+  public signRequest(request: WebResourceLike): WebResource {
```

Comment:
I had to refactor these methods a bit to be able to reuse `getAuthorizationHeader` when hand-crafting the batch request